docs/development/methodology/done.md

```diff
@@ -13,12 +13,12 @@ Currently, we use the following criteria to specify if a feature is
 done:
 
   1. The code for the featured was produced.
-  2. Tests written and passing.
+  2. Tests written and passed.
```

Comment:
'passing' is write here because they continue to pass over time, not just at the moment the tests are tried out.